sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample4_RecognizeEntities.md

```diff
@@ -191,23 +191,60 @@ foreach (RecognizeEntitiesResult entitiesInDocument in entitiesInDocuments)
-            Console.WriteLine("");
+            if (entity.Resolutions.Count > 0)
+            {
+                Console.WriteLine($"    Resolutions:");
+                foreach (BaseResolution resolution in entity.Resolutions)
+                {
+                    // There are several different kinds of resolutions. In this particular sample, we are
+                    // interested in viewing those of type DateTimeResolution and TemporalSpanResolution.
+
+                    if (resolution is DateTimeResolution dateTime)
+                    {
+                        Console.WriteLine($"      Value: {dateTime.Value} ");
+                        Console.WriteLine($"      DateTimeSubKind: {dateTime.DateTimeSubKind} ");
+                        if (!string.IsNullOrEmpty(dateTime.Timex))
+                            Console.WriteLine($"      Timex: {dateTime.Timex}");
+                        if (dateTime.Modifier is not null)
+                            Console.WriteLine($"      Modifier: {dateTime.Modifier}");
+                    }
+
+                    if (resolution is TemporalSpanResolution temporalSpan)
+                    {
+                        if (!string.IsNullOrEmpty(temporalSpan.Begin))
+                            Console.WriteLine($"      Begin: {temporalSpan.Begin}");
+                        if (!string.IsNullOrEmpty(temporalSpan.End))
+                            Console.WriteLine($"      End: {temporalSpan.End}");
+                        if (!string.IsNullOrEmpty(temporalSpan.Duration))
+                            Console.WriteLine($"      Duration: {temporalSpan.Duration}");
+                        if (!string.IsNullOrEmpty(temporalSpan.End))
+                            Console.WriteLine($"      Timex: {temporalSpan.Timex}");
+                        if (temporalSpan.Modifier is not null)
+                            Console.WriteLine($"      Modifier: {temporalSpan.Modifier}");
+                    }
+                }
+            }
+            Console.WriteLine();
         }
 
         Console.WriteLine($"  Document statistics:");
         Console.WriteLine($"    Character count: {entitiesInDocument.Statistics.CharacterCount}");
         Console.WriteLine($"    Transaction count: {entitiesInDocument.Statistics.TransactionCount}");
     }
-    Console.WriteLine("");
+    Console.WriteLine();
 }
 
 Console.WriteLine($"Batch operation statistics:");
 Console.WriteLine($"  Document count: {entitiesInDocuments.Statistics.DocumentCount}");
 Console.WriteLine($"  Valid document count: {entitiesInDocuments.Statistics.ValidDocumentCount}");
 Console.WriteLine($"  Invalid document count: {entitiesInDocuments.Statistics.InvalidDocumentCount}");
 Console.WriteLine($"  Transaction count: {entitiesInDocuments.Statistics.TransactionCount}");
-Console.WriteLine("");
+Console.WriteLine();
 ```
 
+Note that entity resolutions are currently only included in the response when using model version 2022-10-01-preview.
```

Comment:
Might be good to link to the service docs since the model version could change: https://learn.microsoft.com/en-us/azure/cognitive-services/language-service/named-entity-recognition/concepts/entity-resolutions

Reply:
Ah, it's great that they added a note in their own documentation. This has made reconsider having a note in our samples at all. I'm now thinking that it's in our best interest to not mention model versions here and to simply let the documentation from the service team own it.